tests/test_fock.py

```diff
@@ -598,7 +598,7 @@ def test_first_order_cv(self, tol):
         """Test variance of a first order CV expectation value"""
         dev = qml.device("strawberryfields.fock", wires=1, cutoff_dim=15)
 
-        @qml.qnode(dev)
+        @qml.qnode_old.qnode(dev)
```

Comment:
Note: the changes to the new QNode do not seem to be compatible with the GBS device, hence we resort to using the old QNode.